src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -476,8 +475,10 @@ g.test('texture_size,default_value_and_smallest_size,compressed_format')
       })
   )
   .beforeAllSubcases(t => {
-    const { format } = t.params;
+    const { dimension, format } = t.params;
     const info = kTextureFormatInfo[format];
+    // Skip if the dimension is 3D and format is not compatible
+    t.skipIf(dimension === '3d' && !info.texture3D, `${format} is not compatible with 3D texture`);
```

Comment:
Skipping is intended for test cases that work on some systems and not others. Instead, filter this out of the "params" earlier, using `.filter()` or `.unless()`.

Reply:
@kainino0x I tried to address, PTAL! Thank you!